apps/avifenc.c

```diff
@@ -202,6 +202,7 @@ static void syntaxLong(void)
     printf("    -s,--speed S                      : Encoder speed (%d-%d, slowest-fastest, 'default' or 'd' for codec internal defaults. default speed: 6)\n",
            AVIF_SPEED_SLOWEST,
            AVIF_SPEED_FASTEST);
+    printf("                                        libaom & rav1e range 0-10\n");
```

Comment:
libaom range seems to be 0-9:

https://github.com/AOMediaCodec/libavif/blob/db72bb1313a637f9c3aeaf56e3bac43f22254d2d/src/codec_aom.c#L589

Reply:
Done